next/docs/iota-sdk/docs/how-tos/exchange-guide.mdx

```diff
@@ -0,0 +1,262 @@
+
+:::note
+
+If you are unfamiliar with the IOTA SDK, you can find more information in the [documentation](../welcome.md).
+
+:::
+
+You can use the following examples as a guide to implementing _the multi-account approach_ using the `NodeJs` bindings:
+
+1. [Set up the IOTA SDK](#1-set-up-the-iota-sdk).
+2. [Create an account for each user](#2-create-an-account-for-each-user).
+3. [Generate a user address to deposit funds](#3-generate-a-user-address-to-deposit-funds).
+4. [Check the user balance](#4-check-the-account-balance).
+5. [Listen to events](#5-listen-to-events).
+6. [Enable withdrawals](#6-enable-withdrawals).
+
+:::note
+
+If you are looking for other languages, please read the [IOTA SDK overview](../welcome/#your-application-in-the-iota-network).
+
+:::
+
+Since all IOTA SDK bindings are based on core principles provided by the IOTA SDK, the outlined approach is very similar regardless of your chosen programming language.
+
+<details>
+<summary> If you were already supporting Shimmer with wallet.rs </summary>
+
+The [IOTA SDK](https://github.com/iotaledger/iota-sdk) is a brand new project which is a combination and improvement of both [iota.rs](https://github.com/iotaledger/iota.rs) and [wallet.rs](https://github.com/iotaledger/wallet.rs).
+If you were already supporting Shimmer with wallet.rs, we encourage you to switch to the SDK as wallet.rs will be deprecated and not receive any updates anymore.
+
+Most of the APIs are the same and the changes are fairly minimal, here are some of the major changes:
+
+- The npm package changes from `@iota/wallet` to `@iota/sdk`
+- `AccountManager` becomes `Wallet`
+- `manager.generateMnemonic()` becomes `Utils.generateMnemonic()`
+- `generateAddress` becomes `generateEd25519Addresses`
+- `sendAmount` becomes `send`
+- The callbacks to `listen` now provide a properly typed `Event`
+
+</details>
+
+### 1. Set Up the IOTA SDK
+
+First, you should install the components needed to use the IOTA SDK and the binding of your choice; it may vary a bit from language to language. In the case of the [NodeJs binding](../getting-started/nodejs.mdx), it is straightforward since it is distributed via the `npm` package manager.
+
+You can read more about [backup and security in this guide](https://wiki.iota.org/shimmer/introduction/how_tos/backup_security).
+
+```bash
+npm install @iota/sdk dotenv
+```
+
+:::warning
+
+We use `dotenv` to share user-defined constants across many examples. This is for convenience only and shouldn't be done in production.
+
+:::
+
+You can create a `.env` by running the following command:
+
+```bash
+touch .env
+```
+
+The following examples will need these variables to exist, here are some default values you can use but also change:
+
+```bash
+WALLET_DB_PATH="./wallet-database"
+NODE_URL="https://api.testnet.shimmer.network"
+STRONGHOLD_SNAPSHOT_PATH="./wallet.stronghold"
+STRONGHOLD_PASSWORD="*K6%79#yeFn7AMQhMQ"
```

Comment:
If find it better to use a stupid password, so it's more obvious that you should change it, but :shrug:

Reply:
I don't know, I feel like stupid password is not a very good signal to send

Reply:
people might think ... hey ... this a good password already. I'll keep it :trollface: 
also makes it a bit more difficult logging in using the cli-wallet, but okay.

Reply:
okay